unifi/service.go

```diff
@@ -0,0 +1,260 @@
+// Package unifi provides Unifi discovery for RTSP cameras
+package unifi
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/viam-modules/viamrtsp"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/discovery"
+)
+
+const (
+	httpClientTimeout = 30 * time.Second
```

Comment:
timeout is nice. Do you foresee any situations where we might want retries e.g. transient network errors